lib/queues.js

```diff
@@ -163,15 +164,24 @@ class Queue {
   // Push job to queue with the given delay.  Delay can be millisecond,
   // or a string of the form '5s', '2m', etc.
   delayJob(job, duration) {
+    const self = this;
```

Comment:
Some times this is needed, but can be avoided here by using fat-arrow syntax: `.then(jobIDs => {`